IFC4x3/Properties/d/DiscontinuitySpacing_0VQZBIVenAlegXF1tlHC9e/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="DiscontinuitySpacing_0VQZBIVenAlegXF1tlHC9e" Name="DiscontinuitySpacing" UniqueId="1f6a32d2-7e8c-4abe-8aa1-3c1def44c268" PrimaryDataType="IfcLabel" />
```

Comment:
Datatype does not seem to be correct. Referring to the description "perpendicular distance between adjacent discontinuities".

Reply:
Agreed! Shouldn't it be IfcPositiveLengthMeasure?
Ping @j-weil

Reply:
Generally correct, but the Disc Spacing according to ISO 14689 is given in fixed ranges, according to the table above, i.e. predefined classes. We can either keep Ifclabel or define enumerations according to the second row of the table (Less han 20mm, 20 to 60mm, 60 to 200mm,...)

Reply:
I thought about it and think the second is better, becouse then we do not need a "legend" to check what was "very close" etc. In my opinion the second is the better and more common one that needs to be exchanged

Reply:
answereing here on your email, Lars:
Can you check the last comment regarding Pset_GeoObsMappedUnit.DiscontinuitySpacing?
Alternatives:
-	Keep IfcLabel and refer to Table 8 first column
-	Keep IfcLabel and refer to Table 8 second column
-	Use the terms (first column) from Table 8 and create an enum?
**-	Use the Spacing mm (second column) from Table 8 and create an enum?** Jonas: This is the best option.

Reply:
After discussion with @SergejMuhic I changed so we use Table 8 column 1 since the enum typically should be more descriptive in its nature. Column 2 is used as descriptions for the constants.